.github/workflows/tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  python36:
```

Comment:
I have noticed that all tests say Python 3.6, then there is another Python in the grid. Maybe one of the levels can be removed?

Reply:
Looking at an action file I made for another project (https://github.com/dvarrazzo/py-setproctitle/blob/master/.github/workflows/test.yml) probably it's probably just a misnaming of this first level, which can be the like of "linux tests" and leave Python in the grid, as you have.